tests/stm32/src/bin/fdcan.rs

```diff
@@ -0,0 +1,88 @@
+#![no_std]
+#![no_main]
+
+// required-features: fdcan
+
+use defmt::{assert, *};
+use embassy_executor::Spawner;
+use embassy_stm32::peripherals::*;
+use embassy_stm32::{bind_interrupts, can, Config};
+use embassy_time::{Duration, Instant};
+use {defmt_rtt as _, panic_probe as _};
+
+bind_interrupts!(struct Irqs {
+    FDCAN1_IT0 => can::IT0InterruptHandler<FDCAN1>;
+    FDCAN1_IT1 => can::IT1InterruptHandler<FDCAN1>;
+});
+
+#[embassy_executor::main]
+async fn main(_spawner: Spawner) {
+    let config = Config::default();
+
+    let peripherals = embassy_stm32::init(config);
+
+    let mut can = can::Fdcan::new(peripherals.FDCAN1, peripherals.PB8, peripherals.PB9, Irqs);
+
+    // 250k bps
+    can.set_bitrate(250_000);
+
+    let mut can = can.into_external_loopback_mode();
+    //let mut can = can.into_normal_mode();
```

Comment:
This comment and the ones from lines 36, 63 can probably by removed. 

I've got my "lab" soldered and I'll shortly play with adding CAN frame filters. Would be cool to add them then to this test as well, but I don't consider this a requirement.

Reply:
Done. I also figured out that I can add you as a direct collaborator.